hooks/pre-commit

```diff
@@ -2,6 +2,9 @@
 # This file modified from k8s
 # https://github.com/kubernetes/kubernetes/blob/master/hooks/pre-commit
 
+# Now It's removed, The Reason is https://github.com/kubernetes/community/issues/729
```

Comment:
> Now It's removed
Do you mean that we don't need to use this pre-commit hook any more?

Reply:
I think this line just wants to expliain previous line.
Because this file is modified from k8s, and k8s now is removed/

Reply:
https://github.com/pingcap/tidb/pull/9308#discussion_r257182585
yes，you‘re right